dbflute-engine/src/main/java/org/dbflute/logic/doc/decomment/DfDecommentPickupProcess.java

```diff
@@ -27,12 +29,13 @@
  */
 public class DfDecommentPickupProcess {
 
-    private final DfDecoMapFile _decoMapFile = new DfDecoMapFile();
+    private final DfDecoMapFile _decoMapFile = new DfDecoMapFile(() -> LocalDateTime.now());
```

Comment:
Use DBFluteSystem.currentLocalDateTime() instead of LocalDateTime.now() in DBFlute world.